infrastructure/helm/tigerbeetle/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
@rluckom-coil do you think it's worth adding a role and role binding? I don't think tigerbeetle needs to access any of the k8s resources yet.

Reply:
It's a good question. Overall I think it might be best not to. I had initially added a storage class to this PR to set up the PVCs in the way that seemed best. But then I remembered that in our cluster we pretty much have a single "database-y" storage class that it's nice to be able to apply to whatever DBs we end up deploying. And I think it might be that way for a role for this serviceaccount as well--in the cases where it is needed (and I agree with you that I'm not sure what those are yet) I think they might turn out to be environment-specific enough that people deploying this chart would rather supply them themselves.

For another perspective, the cockroach chart's [role gives permission to view and create secrets](https://github.com/cockroachdb/helm-charts/blob/master/cockroachdb/templates/role.yaml) that cockroach uses for client certificates. I think that once tigerbeetle has an auth mechanism (or secure communication between nodes) something like that might become appropriate.

But if you'd rather have them added now for expansion later I can definitely do that.

Reply:
Agreed, let's get this in and take a look at that later.